File: Level_D/d059.cs
using System;

public class Pz{
    public static void Main(){
        var arr = Console.ReadLine().Trim().Split(' ');
        arr[1] = (arr[0] == "J" && arr[1] == "J") ? "Q" : arr[1];
        Console.WriteLine("{0} {1}", arr[0], arr[1]);
    }
}

File: Level_D/Python2/d059.py
# coding: utf-8

if __name__=="__main__":
    arr = raw_input().strip().split(' ')
    if arr[0]=='J' and arr[1]=='J':
        print('J Q')
    else:
        print('{0} {1}'.format(arr[0], arr[1]))
